### ISSUE TITLE
Unable to open pop-up again

### ISSUE THREAD

**Issue by Tralapo** [insites/cookieconsent#357]
Using the basic code from the wizzard on the site, the popup shows up fine on first load. After clicking one of the buttons the pop-up dissapears and I see the little 'Cookie Policy' hiding at the bottom of the screen. This opens the pop-up again.

But this does not seem to work very well. After refreshing the page or browsing around the site, the little button is nowhere to be found.

As it is mandatory to allow users to revoke their consent under GDPR: how can I create a link that calls the pop-up, or make sure the button on the bottom of the screen stays available?

**Comment by NathanCH** [comment]
This package doesn't have a function to pop-up the consent bar again. I managed to do it by triggering a click on the revoke button. Like so:

```
onInitialise: function(status) {
  if (this.options.type === 'opt-out' && !this.hasConsented()) {
    setTimeout(function() {
      document.getElementsByClassName("cc-revoke")[0].click();
    });
  }
},
```

Note setTimeout() to wait for DOM to be accessible.

**Comment by Tralapo** [comment]
I'll explain it a little bit clearer:

- First visit: pop-up shows up (opt-in version) and I can click accept or decline

- After clicking, this little guy is glued to the bottom of the page. Clicking it re-opens the pop-up as it should do. So this package does have a function like that.

![Image](https://i.imgur.com/YUp19Tp.png)

- The button will stay there as long as you stay on the same page, but after going to another page or performing a refresh it's gone and won't come up again. This seems to be a bug.

**Comment by niko5** [comment]
I thin k there are various fixes for this - if you look at this fork it includes the fix https://github.com/niko5/cookieconsent

**Comment by Tralapo** [comment]
It's unclear for me what the fix is.

**Comment by Tralapo** [comment]
@niko5 Using the code from that fork doesn't change anything. After a refresh or changing the page the Cookie Policy button is nowhere to be found.

**Comment by eleumas** [comment]
@Tralapo some news for this issue? me too have this problem...

**Comment by Tralapo** [comment]
Fix it for now by adding this to my CSS:

`.cc-revoke {
display: block !important;
}`

This keeps the Cookie Policy button available (the plugin somehow adds 'style:display:none' to the HTML all the time). It's kind of a hack, but works.

**Comment by eleumas** [comment]
ah ok! i had found this hack...but thanks for your confirm :)

**Comment by neemah** [comment]
So is it a bug? What "revokable" property stands for then?

**Comment by Dolondro** [comment]
Fixed in 3.1.0
